EngineBay.DemoApi/Modules/ModuleRegistration.cs

```diff
@@ -76,7 +71,7 @@ public static WebApplication InitializeDatabase(this WebApplication app)
             var serviceProvider = scope.ServiceProvider;
             var dbInitialiser = serviceProvider.GetRequiredService<DbInitialiser>();
 
-            dbInitialiser.Run(RegisteredModules);
+            dbInitialiser.Run(GetRegisteredModules());
```

Comment:
Same comment as on Engine Bay CE:
Will repeatedly calling the GetRegisteredModules() method not create a new copy of the list with a new instance of each module each time? Would it be better to assign the module list to a private field after the first call and return that saved list each time?